kr8s/_objects.py

```diff
@@ -810,7 +810,12 @@ async def logs(
                 async for line in resp.aiter_lines():
                     yield line
 
-    def portforward(self, remote_port: int, local_port: int = None) -> int:
+    def portforward(
+        self,
+        remote_port: int,
+        local_port: int = None,
+        address: List[str] = ["127.0.0.1"],
```

Comment:
Do you think we could make the type `List[str] | str` so that if the user wants to just pass a single address they can do it with a string?